db/QUICKSTART.md

```
@@ -0,0 +1,56 @@
+docker compose exec postgres psql -U challenge challenge_db -c 'SELECT COUNT(*) FROM sales;'
+
+# Deve mostrar ~500k
+```
+
+## Explore
+
+Explore os dados gerados da forma como quiser e julgar mais eficiente. Navegue pelas tableas e entenda seus relacionamentos.
+
+## Estrutura dos Dados
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)